src/formatDate.js

```diff
@@ -45,12 +45,67 @@
  * @param {string} date
  * @param {string[]} fromFormat
  * @param {string[]} toFormat
- *
+ *formatDate.js
  * @returns {string}
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  let newFormatDate = '';
+  let newDay = '';
+  let newMonth = '';
+  let newYear = '';
+  const newSeparator = toFormat[3];
+
+  let oldDateSplit = [];
```

Comment:
The initialization of 'oldDateSplit' and 'oldYear' is unnecessary since they are immediately given a value in the next lines.